.changeset/clever-files-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```

Comment:
I think this would be a patch, as we also want to fix this memleak earlier for everyone on the current version.
```suggestion
'astro': patch
```

Reply:
Modified.